src/lmflow/models/hf_decoder_model.py

```diff
@@ -241,6 +243,9 @@ def __init__(
         elif tune_strategy == 'adapter':
             raise NotImplementedError('adapter tune strategy not implemented')
 
+        if self.tokenizer.eos_token_id is None:
+            self.tokenizer.eos_token_id = self.backend_model.config.eos_token_id
+        self.tokenizer.pad_token_id = self.tokenizer.eos_token_id
```

Comment:
is it a must to set pad_token_id as eos_token_id?
what if the model has a  special pad_token_id originally?

Could we add `if self.tokenizer.pad_token_ids is None`